### PR TITLE
feat(iOS): enable dampingRatio and springVelocity

### PR DESCRIPTION
**Pull request by Brianggalvez**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-18421

added dampingRatio and springVelocity to allow iOS spring animations

example : 

```javascript
var win = Ti.UI.createWindow({
	backgroundColor: '#fff'
})

var btn = Ti.UI.createButton({
	title: 'Trigger'
})
var box = Ti.UI.createView({
	height: 200,
	width: 200,
	bottom: 30,
	backgroundColor: 'red'
})

btn.addEventListener('click', function() {
	Ti.API.info('Hello world!')
	var scaleMatrix = Ti.UI.create2DMatrix()
	scaleMatrix = scaleMatrix.scale(0.95, 0.95)
	const springAnim = Ti.UI.createAnimation({
		transform: scaleMatrix,
		duration: 400,
		dampingRatio: 0.8,
		springVelocity: 0.4,
		autoreverse: true
	})
	box.animate(springAnim)
})

win.add(btn)
win.add(box);
win.open()
```

### CODE REVIEW

**Comment by vijaysingh-axway** [comment]
@Brianggalvez Thanks for PR. Can you please fix the linting issue?

**Comment by vijaysingh-axway** [comment]
@Brianggalvez Ideally it should work. I have fixed the listing issue.
Every thing looks good. Can you please document these properties? 
Once done we will be good to go with this PR.
Thanks again for PR!

File: apidoc/Titanium/UI/Animation.yml [hunk 1]
```diff
@@ -106,6 +106,11 @@ properties:
     constants: Titanium.UI.ANIMATION_CURVE_*
     platforms: [android, iphone, ipad]
     since: { android: "8.0.0", iphone: "0.9", ipad: "0.9" }
+  
+  - name: dampingRatio
+    summary: Damping ratio of spring. Use a value between 0 and 1. For a smoother deceleration use values closer to 1.
+    type: Number,
+    platforms: [iphone, ipad]
```
Comment (hansemannn) [inline apidoc/Titanium/UI/Animation.yml]: `since: "8.1.0"`

Comment (Brianggalvez) [reply apidoc/Titanium/UI/Animation.yml]: done! thanks!

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

**Review by ssjsamir — APPROVED** [review]
**FR Passed:** Able to see the `ampingRatio` and `springVelocity` being applied to an animation. Tested with the test case mentioned in the description

**Test Environment**

```
iPhone 6 Sim (12.1)
APPC CLI: 7.0.10-14
Operating System Name: Mac OS Mojave
Operating System Version: 10.14.2
Node.js Version: 8.9.1
Xcode 10.1
```

**Comment by jquick-axway** [comment]
It looks like "springVelocity" is really a scale factor, correct?

Would `0.5` run the spring animation at half-speed?
And would `2.0` run the spring animation twice as fast?

**Comment by Brianggalvez** [comment]
Hi Joshua, just prepared a little example on what springVelocity does. It's the view initial velocity so as high it is more bouncing the view does at the end. 


For Android you can find this useful: https://proandroiddev.com/spring-animation-in-android-de3d4197879b

springvelocityexample

**Comment by lokeshchdhry** [comment]
@vijaysingh-axway , I am not able to merge because it is asking for unit test. Can you please do the appropriate.

**Pull request merged**